3d/README.md

```diff
@@ -26,13 +26,13 @@ To see all of the people who are paying attention to our work, see the list of [
 
 If you would like to participate in this group, contact the convener or one of the core members.  
 
-To participate in the communication system of the group, watch the repo and follow the [Issues tracker](https://github.com/tdwg/ac/issues) (look for issues tagged as "3D").  This issues tracker is the mechanism for suggesting specific changes to the Audubon Core standard as well as to raise issues for discussion by the group.
+To participate in the communication system of the group, watch the repo and follow the [Issues tracker](https://github.com/tdwg/ac/issues) (look for issues tagged as "3D").  This issues tracker is the mechanism for suggesting specific changes to the Audiovisual Core standard as well as to raise issues for discussion by the group.
 
 ## Documentation
 
 To see what issues we are currently addressing, see our [Issue Tracker](https://github.com/tdwg/ac/issues).  
 
-Our charter is [here](https://github.com/tdwg/ac/blob/master/3d/charter_3d_task_group_of_audubon_core_2019-06-11.pdf)
+Our charter is [here](https://github.com/tdwg/ac/blob/master/3d/charter_3d_task_group_of_Audiovisual_core_2019-06-11.pdf)
```

Comment:
This is really a historical document and probably should just be left as it is since the "real" current charter is the Task Group landing page on the Community tab of the TDWG website. So it might make more sense to just link to that and not this version.

Reply:
Fixed in 7c67ed2